src/odemis/gui/main.xrc

```diff
@@ -9010,15 +9010,15 @@
                         <border>10</border>
                       </object>
                       <object class="sizeritem">
-                        <object class="wxStaticText">
+                        <object class="wxStaticText" name="tiles_number_y_lbl">
                           <label>Tiles number y</label>
                           <fg>#DDDDDD</fg>
-                          <XRCED>
-                            <assign_var>1</assign_var>
-                          </XRCED>
                           <font>
                             <size>9</size>
                           </font>
+                          <XRCED>
+                              <assign_var>1</assign_var>
+                          </XRCED>
```

Comment:
out of curiosity what does this do (I guess it sets the default value to 1)? And why does it need to be after setting the font size?

Reply:
It tells the xrc-to-python generator to create an attribute on the class with the name of this widget (so that it's easy to access in the code). 
The position doesn't matter. I put it at the end because it was easier for me to check it matches the right XML element (XML is just hard to read).